syntax = "proto3";

package get_data;

service DataEntryGetter {
    rpc GetDataEntry(GetDataEntryRequest) returns (GetDataEntryResponse);
}

message GetDataEntryRequest {
    string dataset_id = 1;
}

message GetDataEntryResponse {
    string data_type = 1;
    string data = 2;
    string entry_id = 3;
    float reward = 4;
    string prelabel = 5;
}